index.html

```diff
@@ -231,6 +231,22 @@
 
 
     // HISTOGRAMS
```

Comment:
I've read through the histogram documentation and the new version example, but I'm having a lot of trouble reconciling the two and identifying where to start with the histogram. I'm trying to work with the example, but I'm getting errors at line 247 that I don't understand at all.